lua/nvim-lsp-installer/servers/fsautocomplete/init.lua

```diff
@@ -0,0 +1,25 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local std = require "nvim-lsp-installer.installers.std"
+local shell = require "nvim-lsp-installer.installers.shell"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "fsharp" },
+        homepage = "https://fsharp.org",
```

Comment:
I think perhaps linking to the language server page would be more in line with how other servers are currently configured:

```suggestion
        homepage = "https://github.com/fsharp/FsAutoComplete",
```

Reply:
😊😊😊